example/isolate-service/README.md

```diff
@@ -0,0 +1,50 @@
+# Description
+The hello world server and client demonstrate how to use Dart gRPC libraries to
```

Comment:
This does not describe this example accurately.
But probably it would be better with a test than an example